web/html/src/core/spa/spa-renderer.ts

```diff
@@ -17,7 +17,7 @@ window.pageRenderers.spa.reactRenderers = window.pageRenderers.spa.reactRenderer
 window.pageRenderers.spa.previousReactRenderers = window.pageRenderers.spa.previousReactRenderers || [];
 
 function addReactApp(appName: string) {
-  getTranslationData();
+  // getTranslationData();
```

Comment:
Can be removed

Reply:
Thanks for the heads up, I missed those.